source/html/index.html

```diff
@@ -125,6 +125,15 @@ <h1 id="bujo-header">R.I.V.E.R BuJo</h1>
 
     <footer>
 
+        <!-- <details>
```

Comment:
As a group we may want to start considering removing commented out code in our PR's next sprint (it may help graders know what has been finalized in our code)